modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a Route via an Ingress
```

Comment:
s/Creating a Route via an Ingress/Creating a route through an Ingress object

Does this change make sense in context? The IBM style guide advises against using "via": "Use this term, which means 'by way of', to refer to routing data, such as in 'Messages are routed via
SMTP.' In other situations, try to replace 'via' with one of the following synonyms: 'across', 'along', 'by', 'from', 'on', or 'through'."

Reply:
`by way of` is pretty much what I mean, but metaphorically, not literally. `Through` sounds good as well.

I've changed it to `through`. Since I am not a native speaker I am probably not the best candidate to find the best wording for this :upside_down_face: